frontend/src/pages/AccountSend.vue

```diff
@@ -397,6 +399,18 @@ function useSendForm() {
 
     // Validate the form
     await sendFormRef.value?.validate();
+
+    // Switch chain
+    const chain = supportedChains.filter((chain) => chain.chainId === `0x${Number(linkChainId)?.toString(16)}`);
```

Comment:
This is probably a bit more robust/safe, I don't really like trusting native JS type/format conversions for important things 😅
```suggestion
    const chain = supportedChains.filter((chain) => chain.chainId === BigNumber.from(linkChainId).toHexString;
```

Reply:
I ended up using hexValue instead of toHexString because it does not have leading 0s.